pyupgrade.py

```diff
@@ -1084,6 +1085,27 @@ def visit_Call(self, node):  # type: (ast.Call) -> None
                 not _starargs(node)
         ):
             self.native_literals.add(_ast_to_offset(node))
+        elif (
+                isinstance(node.func, ast.Attribute) and
+                isinstance(node.func.value, ast.Str) and
+                node.func.attr == 'encode' and
+                not _starargs(node) and
+                len(node.args) == 1 and
+                isinstance(node.args[0], ast.Str) and
+                node.args[0].s.lower() == 'utf-8'
+        ):
+            self.encode_calls[_ast_to_offset(node)] = node
+        elif (
+                isinstance(node.func, ast.Attribute) and
+                isinstance(node.func.value, ast.Str) and
+                node.func.attr == 'encode' and
+                not _starargs(node) and
+                len(node.keywords) == 1 and
+                node.keywords[0].arg == 'encoding' and
+                isinstance(node.keywords[0].value, ast.Str) and
+                node.keywords[0].value.s.lower() == 'utf-8'
+        ):
```

Comment:
I wouldn't bother with this `if`, I don't think I've ever seen `.encode(encoding='UTF-8')`

Reply:
OK